go/registry/api/runtime.go

```diff
@@ -256,6 +263,9 @@ type Runtime struct { // nolint: maligned
 	// Storage stores parameters of the storage committee.
 	Storage StorageParameters `json:"storage,omitempty"`
 
+	// KeyManager parameters stores paratemers specific to Keymanager runtime.
+	KeyManagerParameters KeyManagerParameters `json:"key_manager_parameters,omitempty"`
```

Comment:
Would rather just use `KeyManager` (as we do for other types of parameters), but we already use it for specifying keymanager runtime id. 

Since this is already "breaking" could also rename the `KeyManager` field to `KeyManagerID` here, and use `KeyManager` for parameters.

Reply:
I would just rename the field tbh.

Reply:
Yeah, just rename the field.